coalib/results/Diff.py

```diff
@@ -523,12 +542,12 @@ def modify_line(self, line_nr, replacement):
         if linediff.change is not False and linediff.change[1] != replacement:
             if len(replacement) == len(linediff.change[1]) == 1:
                 raise ConflictError('Cannot merge the given line changes.')
-
+            # strings are used here instead of lists.
```

Comment:
When a line is modified twice, it is possible for the contents to be merged.
If that is the case, in the lines below 545 new diffs are generated from the changes lines. Since these diffs are not generated from a list, but instead from a string, the elements of these diffs will be chars. 
`diff = Diff('hello')`
 => `diff._file = ['h','e','l','l','o']`
These will be added and formed to a new replacement line. 
Its quite interesting actually, but a bit confusing if you don't know these diffs are created from strings, so I thought I add a comment.